### DISCUSSION TITLE
2023년 03월 18일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#8] [General]
# 2023년 03월 18일

⚖️ 공부한 것, 일기를 적는 개인 위키입니다 인덱스 페이지: [ wiki-index ] 이번 달의 TIL: [ 2023-03|2023년 3월의 TIL✨ ] 현재 자주 쓰는 글들: [ wooteco|우아한테크코스 ]

https://cyma-s.github.io/2023-03-18/

**Comment by drunkenhw** [comment]
죄송합니다. 밤에 체스 두자고 안하겠습니다.

**Reply by Cyma-s** [reply]
체스 둘 시간은 있습니다 ㅋㅋ
